srcpkgs/libknet1/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'libknet1'
+# only library and development files now, if interested in kronosnetd update!
+pkgname=libknet1
+version=1.11
+revision=1
+wrksrc=kronosnet-${version}
+build_style=gnu-configure
+configure_args="--disable-crypto-nss --disable-dependency-tracking
+ --disable-libknet-sctp --disable-libnozzle --disable-kronosnetd --disable-poc
+ --disable-static"
+hostmakedepends="autoconf automake doxygen glibc-devel icu-devel libqb-devel
+ libtool libxml2-devel pkg-config"
+makedepends="bzip2-devel liblz4-devel liblzma-devel libqb-devel libressl-devel
+ libxml2-devel libzstd-devel lzo-devel zlib-devel"
+short_desc="Kronosnet core switching implementation (libraries)"
+maintainer="Jiří Bělka <jirib79@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.kronosnet.org"
+distfiles="https://kronosnet.org/releases/kronosnet-${version}.tar.bz2"
+checksum=c2ee20a9b5b0791ed5379b38b0b51d549603933925a839f045455553f03e6114
+
+post_install() {
+	rm -rf ${DESTDIR}/usr/share/doc/kronosnet
+	vdoc README
+}
+
+libknet1-devel_package() {
+	license="LGPL-2.1-only"
+	short_desc="${short_desc} (development files)"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.so"
+		vmove "usr/lib/pkgconfig"
+		vmove usr/share/man/man3
+		mkdir -p ${PKGDESTDIR}/usr/share/doc/libknet1-devel
+		cp README ${PKGDESTDIR}/usr/share/doc/libknet1-devel
```

Comment:
No need to duplicate the README, `libknet1-devel` should depend on `libknet1>=${version}_${revision}`